design/src/main/kotlin/uk/govuk/app/design/ui/component/List.kt

```diff
@@ -82,7 +82,6 @@ fun ExternalLinkListItem(
     ) {
         Row(
             modifier = Modifier.padding(applyPadding(subText)),
```

Comment:
Redundant comma here (only mentioning as not causing any issues). There's another one on line 89 also if you do decide to remove this one.

Reply:
Removed